widgetastic_manageiq/vm_reconfigure.py

```diff
@@ -30,3 +30,21 @@ def click_add_disk(self):
     def click_cancel_add(self):
         """Clicks the Cancel Add button to cancel adding a new row"""
         self.cancel_add_btn.click()
+
+
+class NetworkAdaptersTable(VanillaTable):
+    """Table to add and remove Disks (in VM Reconfigure form)"""
+
+    add_nw_btn = DisksButton("contains", "Add Network", classes=[Button.PRIMARY])
+    cancel_add_nw_btn = DisksButton("contains", "Cancel Add", classes=[Button.DEFAULT])
+
+    def click_add_nw_adapter(self):
+        """Clicks the Add Network Adapter button attached to the table and
+        returns the new editable row
+        """
+        self.add_nw_btn.click()
+        return self[-1]
+
+    def click_cancel_add_nw_adapter(self):
+        """Clicks the Cancel Add button to cancel adding a new row"""
```

Comment:
"""Clicks the Cancel  button to cancel adding a new row"""

Reply:
the name of button is `Cancel Add`